### ISSUE TITLE
Feature Request: Email Alert

### ISSUE THREAD

**Issue by COLABORATI** [johngodley/redirection#130]
One very obvious missing feature is that it is not possible to receive email alerts on 404 errors, what can be very important. Or am I missing something and this feature exists?

**Comment by AndrewKvalheim** [comment]
For a quick solution I've just been emitting the log from a cron job:

```shell
wp eval 'RE_404::export_to_csv();' | tail -n+2
```

**Comment by fharper** [comment]
The log doesn't include the 404s, no?

**Comment by johngodley** [comment]
This is referring to the 404 log.

**Comment by fharper** [comment]
@johngodley : yep, but I don't have any RSS button option in the 404s tab like I have in the log one.

**Comment by johngodley** [comment]
Ok, can you create a separate issue for it? This one is 3 years old and unrelated.

**Comment by fharper** [comment]
Yep, sorry!
